logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/webSocketStatus/EnableWebSocketUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.wire.kalium.logic.feature.user.webSocketStatus
+
+import com.wire.kalium.logic.StorageFailure
+import com.wire.kalium.logic.configuration.UserConfigRepository
+import com.wire.kalium.logic.functional.Either
+
+interface EnableWebSocketUseCase {
+    operator fun invoke(enabled: Boolean): Either<StorageFailure, Unit>
```

Comment:
We should not expose `Either` to clients, so I think we need to wrap this in a `Result` class